packages/ui/src/components/page/Sidebar/CurrentMember/CurrentMember.tsx

```diff
@@ -10,19 +10,24 @@ import { AddMembershipButton } from '../../../membership/AddMembershipButton'
 import { SwitchMemberModal } from './SwitchMemberModal'
 
 export const CurrentMember = () => {
-  const { count, members, active } = useMyMemberships()
+  const { count, members, active, setActive } = useMyMemberships()
   const [isOpen, toggleOpen] = useToggle()
   const [isCreateOpen, toggleCreateOpen] = useToggle()
+  useEffect(() => {
+    count == 1 && setActive(members[0])
+  }, [count])
 
   if (count < 1) {
     return <AddMembershipButton>Create a membership</AddMembershipButton>
   }
 
+  count > 1 && !active && !isOpen && toggleOpen()
```

Comment:
This one looks odd. I think that we could do something better:

*   I'd move that out of "render" zone, so right after the last `useEffect()`.
*   The other thing is that I'm not sure if we have to call this on every render – I'm thinking about `useEffect()` hook for that
*   Anything more complex than single `&&` deserves own variable or own `if` especially in "logic" part of code.